runtime/frequency/src/apis.rs

```diff
@@ -149,11 +152,13 @@ impl_runtime_apis! {
 		}
 
 		fn get_preset(id: &Option<sp_genesis_builder::PresetId>) -> Option<Vec<u8>> {
-			get_preset::<RuntimeGenesisConfig>(id, |_| None)
+			get_preset::<RuntimeGenesisConfig>(id,  &genesis_config_presets::get_preset)
 		}
 
 		fn preset_names() -> Vec<sp_genesis_builder::PresetId> {
-			Default::default()
+			vec![
+				sp_genesis_builder::PresetId::from("development"),
```

Comment:
my guess is that you are currently hardcoding `development` in here but it should support mainnet and testnet in future PR also?

Reply:
That’s correct. To make it easier to review, I’m breaking each one into its own PR.